docs/operation/cluster_and_service_configuration.rst

```diff
@@ -0,0 +1,206 @@
+Cluster and Services Configurations
```

Comment:
SOo how to configure Alert notification? or add a new rule?

Reply:
Under the `Managing Monitoring` section of this doc, I am going to add a `Todo` and then reference the EPIC https://github.com/scality/metalk8s/issues/2193 since it is stated within the EPIC scope.

Let me know if we should do otherwise!

Reply:
I am okay to do it under the other EPIC but remember it is the most important need, so let's do it right after.